multivac/gather_data.py

```diff
@@ -105,6 +106,16 @@ def github_time_to_unix(time: str) -> float:
     return time_to_unix
 
 
+# https://github.com/tarantool/tarantool/tree/master/.github/workflows
+OS_MATCHER = re.compile(r"[a-z]+_[0-9]+(_[0-9]+)?")
+FREEBSD_MATCHER = re.compile(r"freebsd-[0-9]{2}")
+# Jobs running right on the runner, not in a container
+NON_CONTAINERIZED_WORKFLOWS = re.compile(
```

Comment:
OTHER_WORKFLOWS_MATCHER?

Reply:
Not just any other workflows, but ones running on default runners. Maybe, DEFAULT_RUNNERS_MATCHER?

Reply:
Works for me.